packages/frontend/bin/build-frontend-ts.mjs

```diff
@@ -26,7 +26,7 @@ function config(options) {
   return {
     entryPoints: ['src/main.tsx'],
     bundle: true,
-    minify: isMinify,
+    minify: false,
```

Comment:
Why? `isMinify` is unused now.

Reply:
good catch a debugging hack that was left in